test/test_torch.py

```diff
@@ -14459,9 +14458,10 @@ def matrixmultiply(mat1, mat2):
         for (n, m, p) in [(20, 10, 5), (15, 5, 10), (5, 18, 10)]:
             _test_mm(n, m, p, torch.float32, lambda x, y: torch.randn(x, y, dtype=torch.float32, device=device))
             _test_mm(n, m, p, torch.float64, lambda x, y: torch.randn(x, y, dtype=torch.float64, device=device))
-            _test_mm(n, m, p, torch.int32, lambda x, y: torch.randint(0, 100, (x, y), dtype=torch.int32, device=device))
-            _test_mm(n, m, p, torch.int64, lambda x, y: torch.randint(0, 100, (x, y), dtype=torch.int64, device=device))
-            _test_mm(n, m, p, torch.bfloat16, lambda x, y: torch.randn(x, y, dtype=torch.float32, device=device).bfloat16())
+            if device == 'cpu':
```

Comment:
this probably could be done by adding dtype argument to the test, and adding @dtypes, @dtypesIfCUDA wrappers

Reply:
Done